### PR TITLE
Bitpacked array doesn't have patches if they're all null values

### PR DESCRIPTION
**Pull request by robert3005**

### CODE REVIEW

**Comment by robert3005** [comment]
~working on a test~ Done

File: encodings/fastlanes/src/bitpacking/compress.rs [hunk 1]
```diff
@@ -96,22 +98,27 @@ pub fn bitpack_patches(
...
-        ).unwrap().into_array()
+
+        (!indices.is_empty()).then(|| {
+            SparseArray::try_new(
+                indices.into_array(),
+                PrimitiveArray::from_vec(values, Validity::AllValid).into_array(),
+                parray.len(),
+                Scalar::null(parray.dtype().as_nullable()),
+            )
+            .unwrap()
```
Comment (lwwmanning) [inline encodings/fastlanes/src/bitpacking/compress.rs]: no unwrapping! (how was this not caught by clippy?)

Comment (robert3005) [reply encodings/fastlanes/src/bitpacking/compress.rs]: hahaha, no clue. It was like this before fwiw

Comment (robert3005) [reply encodings/fastlanes/src/bitpacking/compress.rs]: ok, fixed

**Review by lwwmanning — APPROVED** [review]

**Pull request merged**